src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj

```diff
@@ -24,6 +24,8 @@
     <NoDocumentationFile>true</NoDocumentationFile>
     <UseCommonOutputDirectory>true</UseCommonOutputDirectory>
     <ProjectSystemLayer>VisualStudio</ProjectSystemLayer>
+    <NeedsSigningToRelease>true</NeedsSigningToRelease>
+    <ProducingSignedVsix>true</ProducingSignedVsix>
```

Comment:
We also have the ProjectSystemDogfoodSetup Vsix which we are not shipping and should not be signed. Also this tells us that this is a vsix. I could have used one of the properties that we usually see only in vsix but then over a period of time that property could change and its meaning could change. By having a dedicated property, it is much easier to track changes and reasoning behind using a property